lib/diagnostics_channel.js

```diff
@@ -65,7 +65,7 @@ function markActive(channel) {
 
 function maybeMarkInactive(channel) {
   // When there are no more active subscribers or bound, restore to fast prototype.
-  if (!channel._subscribers.length && !channel._stores.size) {
+  if ((!channel._subscribers.length && !channel._stores.size) || channel._subscribers.every((v) => v === undefined)) {
```

Comment:
Primordials?

Reply:
I changed the code and it's now irrelevant.